recipe/meta.yaml

```diff
@@ -62,6 +79,8 @@ outputs:
       host:
         - {{ pin_subpackage("rust-std-" ~ rust_arch, exact=True) }}
         - {{ compiler('c') }}  # [linux] -- rustc needs a toolchain to link executables
+        # zlib is linked by **/lib/libLLVM-*-rust-*.so
+        - zlib  # [linux]
```

Comment:
We could also make this `# [unix]`, but I'm unsure what is preferred on macOS. We get
```
Needed DSO /Applications/Xcode_12.4.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk/usr/lib/libz.1.dylib found in $SYSROOT
```
on that platform.

Reply:
We actually prefer our own package on macOS too